build.py

```diff
@@ -641,6 +664,22 @@ def build(self):
 
 Project.add(Project_glib_openssl())
 
+class Project_graphene(GitRepo, Meson):
+    def __init__(self):
+        Meson.__init__(self,
+            'graphene',
+            repo_url = 'https://github.com/ebassi/graphene',
+            fetch_submodules = False,
+            tag = None,
+            dependencies = ['ninja', 'meson', 'pkg-config', 'glib'],

```

Comment:
Graphene can be build without pkg-config (obviously without gobject support) so imho it make sense to have it when needed (even if the libraris that we'll use probably are something glib/gtk related and so we'll put it in everyone).

For ninja you can create, from meson, a visual studio 2015 solution/project and use it to do everything so also in this case I think that saying that we use ninja is correct.

Besides this is the first project using meson and ninja, we can wait to have other two or three and see what we have to put to make everything works: at the end if we use meson only with ninja and for the gtk stuff putting pkg-config 6 ninja as meson dependencies is really useful.